containers/jml-extract-lambda/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2024-01-29
+
+### Changed
+
+- Change the handler to updated the secret it needs to change the image as well
```

Comment:
Hi,

Can you update this to be a clearer description of what this update adds? I'd be sure from reading this

Reply:
upadted